### ISSUE TITLE
Running pretrained model in pytorch

### ISSUE THREAD

**Issue by czhu95** [jiecaoyu/XNOR-Net-PyTorch#39]
Hi jiecaoyu,

I was trying to run the pretrained AlexNet model you provided but failed. I noticed that you used an older version of pytorch which makes the network arch incompatible with 3.0. So I switched to 2.0.

I also noticed that you read all the images from caffe generated lmdb files. I instead use pytorch ImageFolder to read raw images and resize them to 256 by 256. I followed the same normalization transformation as subtracting per-pixel imagemean. So after transformation data lies roughly between -128 and +128.

Is there anything else that I missed? I really appreciate your help on reproducing your results!

Thanks.

**Comment by jiecaoyu** [comment]
Hi @czhu95 it should achieve a reasonable accuracy. What's the accuracy you can get now? I am also interning at FB, feel free to send me internal messages.
